examples/caliper/CMakeLists.txt

```
@@ -0,0 +1,51 @@
+# Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+# at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+# LICENSE and NOTICE for details. LLNL-CODE-806117.
+#
+# This file is part of the MFEM library. For more information and source code
+# availability visit https://mfem.org.
+#
+# MFEM is free software; you can redistribute it and/or modify it under the
+# terms of the BSD-3 license. We welcome feedback and contributions, see file
+# CONTRIBUTING.md for details.
+
+set(CALIPER_EXAMPLES_SRCS)
+
+list(APPEND CALIPER_EXE_SRCS
+  ex1.cpp
+  )
+
+if (MFEM_USE_MPI)
+list(APPEND CALIPER_EXE_SRCS
+    ex1p.cpp
+    )
+endif()
+
+
+# Include the source directory where mfem.hpp and mfem-performance.hpp are.
+include_directories(BEFORE ${PROJECT_BINARY_DIR})
+
+# Add one executable per cpp file
+set(PFX caliper_)
+add_mfem_examples(CALIPER_EXE_SRCS ${PFX})
```

I would prefer the full word here
```suggestion
set(PREFIX caliper_)
add_mfem_examples(CALIPER_EXE_SRCS ${PREFIX})
```

---

Agreed